src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -436,20 +440,32 @@ def sample_prior(
                     init_file = config_node.get_init_file_fmt()
                     if "%d" in init_file:
                         init_file = init_file % realization_nr
-                    grid_file = ensemble.experiment.grid
-                    assert grid_file is not None
-                    grid = xtgeo.grid_from_file(grid_file)
-                    try:
-                        props = xtgeo.gridproperty_from_file(
-                            init_file, name=parameter, grid=grid
-                        )
-                    except PermissionError as err:
-                        context_message = (
-                            f"Failed to open init file for parameter {parameter!r}"
+
+                    grid = ensemble.experiment.grid
+                    assert grid
```

Comment:
Do we want to assert with an error message or do you think this is fine as is?

Reply:
Ill remove the asserts before merging